packages/wonder-blocks-layout/components/media-layout.js

```diff
@@ -37,11 +38,11 @@ type Props = {|
      *                  stylesheet for this mediaSize (as specified in the
      *                  styleSheets prop).
      */
-    children: ({
+    children: ({|
```

Comment:
You moved `ChildrenArgs` out above, do you want to do that here, too? I guess I like leaving them inline as then it's more obvious what the args are inside the Styleguidist docs.

Reply:
VSCode got confused.  I've reverted this change.